### DISCUSSION TITLE
Building and submitting bottles manually

### DISCUSSION THREAD

**Discussion by whitebear60** [Homebrew/discussions#2928] [Tap maintenance and Homebrew development] [answered]
Good morning.

As far as I know, the formulae bottles is building in Homebrew cloud with GitHub runners for the three most recent macOS versions.
Can I build bottles with `brew install --build-bottle <formula> && brew bottle <formula>` on macOS Mojave and then submit it to the `homebrew/core` repository?

**Comment by carlocab** [comment, marked as answer]
Thanks very much for your kind offer, but there are security issues that stem from redistributing binaries we didn't build ourselves that we'd like to avoid.

You might be interested in https://github.com/gromgit/homebrew-core-mojave though.

**Reply by whitebear60** [reply]
That's that I expected to see, have read about this before in one place, but didn't remember where exactly... Thanks anyway, will continue maintaining `homebrew/cask` versions to load older versions of software on older macOS versions (latest supported version for each macOS version, as specified by developers of the software)

**Reply by gromgit** [reply]
As the maintainer of the project @carlocab mentioned, I'm definitely not accepting third-party bottles either, and for the same (security) reasons.

**Reply by whitebear60** [reply]
@gromgit That is expected too. Anyway, tried to use your fork of home-brew/core, but it didn't work at all for me, opened an issue in your repo. Thanks
